Tests/UnitTests/Core/Geometry/ProtoLayerTests.cpp

```diff
@@ -111,6 +130,10 @@ BOOST_AUTO_TEST_CASE(ProtoLayerTests) {
   CHECK_CLOSE_ABS(protoLayerRot.max(binZ), 6., 1e-8);
   CHECK_CLOSE_ABS(protoLayerRot.min(binR), 3., 1e-8);
   CHECK_CLOSE_ABS(protoLayerRot.max(binR), std::sqrt(3 * 3 + 6 * 6), 1e-8);
+
+  std::stringstream sstream;
+  protoLayerRot.toStream(sstream);
+  std::cout << sstream.str() << std::endl;
```

Comment:
This would be a nice thing to test whether it contains the information of interest or at least to improve the code coverage. However while looking up the content, it looks like it just prints a fixed string without any information at all.